### PR TITLE
Add zsh examples to environment docs (and simplify shell)

### PR DESCRIPTION
**Pull request by dcramer**
- Remove subjective "how to reload shell" (let folks figure out their own edge cases)
- Add cold-folded zsh examples
- Simplify a few language bits

### CODE REVIEW

File: src/docs/environment.mdx [hunk 1]
```diff
@@ -35,22 +31,28 @@ To install the required versions of Python you'll need to run `pyenv install`. T
...
+````
 
-```bash
+Configure your shell to load pyenv:
+
+```bash {filename: ~/.bashrc}
+eval "$(pyenv init -)"
+```
+
+```zsh {filename: ~/.zshrc}
```
Comment (mitsuhiko) [inline src/docs/environment.mdx:47]:
```suggestion
```zsh {filename: ~/.zshrc}{tabTitle: ZSH}
```

File: src/docs/environment.mdx [hunk 2]
```diff
@@ -83,10 +85,11 @@ curl https://get.volta.sh | bash
 The volta installer will tell you to "open a new terminal to start using Volta", but you don't have to! You can just reload your shell like before:
 
 ```bash
-PATH="" exec /bin/bash -l
```
Comment (joshuarli) [inline src/docs/environment.mdx:86]: lol rip

**Comment by joshuarli** [comment]
bit late to the party but lgtm

File: src/docs/environment.mdx (hunk 2 above)
Comment (dcramer) [reply src/docs/environment.mdx:86]: yeah i figure if people actually know wtf is going on here they likely can fix it themselves (or just reopen shells)

did this to simplify things, and I think some of this is "ya'll gotta learn how shells work"

Comment (dcramer) [reply src/docs/environment.mdx:86]: (asside probably more simplification we can do here, still need to get collapsed blocks in tho)

**Pull request merged**